src/main/java/minicraft/screen/QuestsDisplay.java

```diff
@@ -345,12 +341,9 @@ public static boolean isQuestDone(String id) {
 	public static Object getQuestData(String id) {
 		return questStatus.get(id);
 	}
-	public static void setQuestData(String id, QuestStatus val) {
+	public static void setQuestData(String id, Object val) {
```

Comment:
Json is probably better in this case, or maybe CBOR.

Reply:
I will adjust that when we need to use QuestData.

Reply:
Yup